src/libraries/System.Private.CoreLib/src/System/Reflection/AssemblyNameFormatter.cs

```diff
@@ -99,24 +99,32 @@ private static void AppendQuoted(this StringBuilder sb, string s)
 
             for (int i = 0; i < s.Length; i++)
             {
-                bool addedEscape = false;
-                foreach (KeyValuePair<char, string> kv in EscapeSequences)
+                switch (s[i])
                 {
-                    string escapeReplacement = kv.Value;
-                    if (!(s[i] == escapeReplacement[0]))
+                    case '\\':
+                    case ',':
+                    case '=':
+                    case '\'':
+                    case '"':
+                        sb.Append('\\');
+                        break;
+                    case '\t':
+                        sb.Append('\\').Append('t');
```

Comment:
Is this faster than `sb.Append("\\t")`?

Reply:
It's probably same now when Append is has a fast path for 2 characters string (updating)